tests/cypress/integration/actions_tasks_objects/case_15_group_features.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const';
+
+context('Group features', () => {
+    const caseId = '15';
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        switchLabel: false,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    };
+    const createRectangleShape2PointsSecond = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        switchLabel: false,
+        firstX: createRectangleShape2Points.firstX + 300,
+        firstY: createRectangleShape2Points.firstY,
+        secondX: createRectangleShape2Points.secondX + 300,
+        secondY: createRectangleShape2Points.secondY,
+    };
+    const createRectangleTrack2Points = {
+        points: 'By 2 Points',
+        type: 'Track',
+        switchLabel: false,
+        firstX: 250,
+        firstY: 600,
+        secondX: 350,
+        secondY: 700,
+    };
+    const createRectangleTrack2PointsSecond = {
+        points: 'By 2 Points',
+        type: 'Track',
+        switchLabel: false,
+        firstX: createRectangleTrack2Points.firstX + 300,
+        firstY: createRectangleTrack2Points.firstY,
+        secondX: createRectangleTrack2Points.secondX + 300,
+        secondY: createRectangleTrack2Points.secondY,
+    };
+
+    let fillChangeAppearance = '';
+    let fillGroupedShape = '';
+    let fillGroupedTrack = '';
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Create two shapes and two tracks.', () => {
+            cy.createRectangle(createRectangleShape2Points);
+            cy.createRectangle(createRectangleShape2PointsSecond);
+            cy.createRectangle(createRectangleTrack2Points);
+            cy.createRectangle(createRectangleTrack2PointsSecond);
+        });
+        it('Set option "Color by" to "Group".', () => {
+            cy.changeAppearance('Group');
+            cy.get('.cvat_canvas_shape').then($listCanvasShapes => {
+                for (let i=0; i<$listCanvasShapes.length; i++) {
+                    cy.get($listCanvasShapes[i]).should('have.css', 'fill').then($fill => {
+                        // expected rgb(224, 224, 224) to equal rgb(224, 224, 224)
+                        expect($fill).to.be.equal($fill);
```

Comment:
``expect($fill).to.be.equal($fill);``? How does it work?

Reply:
Yes, this is not how it should be. Thank you, corrected. Here I make a check that the checked parameter is the same for all elements in the loop